ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ranger/RangerRestClientImpl.java

```diff
@@ -444,7 +482,7 @@ boolean checkConnectionPlain(String url, HiveConf hiveConf) {
     List<RangerPolicy.RangerPolicyItemAccess> denyExceptionsPolicyItemAccesses = new ArrayList<RangerPolicy.
         RangerPolicyItemAccess>();
 
-    resourceNameList.add(sourceDb);
+    resourceNameList.add(targetDb);
     resourceNameList.add("dummy");
```

Comment:
is this dummy needed?

Reply:
It is required to avoid new deny policy from overriding the existing policy. Ref. HIVE-24371.